src/components/ForestTypeModal/ForestTypeDescription/so/ForestTypesDescription.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React, { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Document, Page } from 'react-pdf/dist/esm/entry.webpack';
```

Comment:
Is there a way to rerender the PDF on display resize? Because on mobile portrait view the text is very small and when you switch to landscape currently you need to reload the page for the PDF to use up the full width.

Reply:
I resolved this